### PR TITLE
Deprecate unused private service RPCs until we port to package:vm_service

### PR DESCRIPTION
**Pull request by bkonyi**

### CODE REVIEW

File: packages/devtools/lib/src/vm_service_wrapper.dart [hunk 1]
```diff
@@ -186,10 +186,10 @@ class VmServiceWrapper implements VmService {
     );
   }
 
+  @deprecated
   @override
   Future<CpuProfile> getCpuProfile(String isolateId, String tags) {
-    return _trackFuture(
-        'getCpuProfile', _vmService.getCpuProfile(isolateId, tags));
+    throw UnsupportedError('Private service RPC');
```
Comment (devoncarew) [inline packages/devtools/lib/src/vm_service_wrapper.dart]: I think we're calling this method (`getCpuProfile()`) in the app currently. Perhaps change to using `callMethod()` here?

Comment (bkonyi) [reply packages/devtools/lib/src/vm_service_wrapper.dart]: We're actually calling `getCpuProfileTimeline` in `getCpuProfile` in `CpuProfilerService`'s. I was confused at first as well, but the analyzer doesn't seem to think this is used anywhere else (gave me a deprecation hint when I originally tried to deprecate `clearCpuProfile`).

**Comment by bkonyi** [comment]
> As long as `collectAllGarbage`, `requestHeapSnapshot`, and `getCpuProfile` are unused by the current code, lgtm.
> 
> I'd wait to land this until we figure out the breakages on the CI (likely related to new versions of build_runner and related packages).

Looks like they've cleared up now. I'll merge this now, but feel free to revert if there's any concerns.

File: packages/devtools/lib/src/vm_service_wrapper.dart [hunk 2]
```diff
@@ -99,8 +99,8 @@ class VmServiceWrapper implements VmService {
 
   @override
   Future<Success> clearCpuProfile(String isolateId) {
-    return _trackFuture(
-        'clearCpuProfile', _vmService.clearCpuProfile(isolateId));
+    return _trackFuture('clearCpuProfile',
+        callMethod('_clearCpuProfile', isolateId: isolateId));
   }
```
Comment (kenzieschmoll) [inline packages/devtools/lib/src/vm_service_wrapper.dart]: Is this backwards compatible with all possible versions of the vm_service users will be on?

Comment (bkonyi) [reply packages/devtools/lib/src/vm_service_wrapper.dart]: At this point, yes. No changes have been made to the `_clearCpuProfile` RPC. This change is just to allow for the removal of the `clearCpuProfile` method in `package:vm_service` as it's currently undocumented.

Comment (kenzieschmoll) [reply packages/devtools/lib/src/vm_service_wrapper.dart]: So once we are ported to package:vm_service, will `clearCpuProfile` again become a public RPC? I guess I'm unclear why we are going back to calling the private RPC here.

Comment (bkonyi) [reply packages/devtools/lib/src/vm_service_wrapper.dart]: This was never a public RPC, so this is effectively a no-op from a functionality perspective. Maybe you're thinking of [clearVMTimeline](https://github.com/dart-lang/sdk/blob/master/runtime/vm/service/service.md#clearvmtimeline)?

**Pull request merged**